.prettierrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
Prettier settings: https://prettier.io/docs/en/options.html

The settings here are mostly defaults, although I did adjust `printWidth` and `singleQuote`. I found that using the default `printWidth` of 80 made for odd results in Vue templates. Setting `singleQuote` to true is my preference and matches our existing code.

I'm happy to alter the settings if others don't agree; the whole point is to let the formatter worry about it.